Frontend/src/components/GoogleLogin.jsx

```diff
@@ -17,12 +17,28 @@ export const GoogleButton = () => {
                     return response.json();
                 })
                 .then((userData) => {
-                    console.log(userData);
+                    localStorage.setItem(
+                        "userName",
+                        userData ? userData.name : ""
+                    );
+                    localStorage.setItem(
+                        "userEmail",
+                        userData ? userData.email : ""
+                    );
+                    localStorage.setItem(
+                        "userImg",
+                        userData ? userData.picture : ""
+                    );
+
+                    setTimeout(() => {
+                        const redirect_uri = "http://localhost:5173/inputLanguage";
```

Comment:
Add `http://localhost:5173` to `.env` and use `process.env` here

Reply:
sure